terraform/scripts/generate_vpc_peering_test.go

```diff
@@ -0,0 +1,196 @@
+package scripts_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+var _ = Describe("VPC peering", func() {
+	type PeerConfig struct {
+		PeerName   string `json:"peer_name"`
+		AccountID  string `json:"account_id"`
+		VpcID      string `json:"vpc_id"`
+		SubnetCIDR string `json:"subnet_cidr"`
```

Comment:
Gah! This is still not validating the case of the fields in the input. From [the docs](https://golang.org/pkg/encoding/json/#Unmarshal):

> To unmarshal JSON into a struct, Unmarshal matches incoming object keys to the keys used by Marshal (either the struct field name or its tag), preferring an exact match but also accepting a case-insensitive match. Unmarshal will only set exported fields of the struct.

I think we'll need to use a `map[string]string` to achieve case validation. At least we can use [Gomega's collection assertions](http://onsi.github.io/gomega/#working-with-collections) for this (`Expect(foo).To(HaveKey("bar"))`, or even `Expect(foo).To(HaveKeyWithValue("bar", Not(BeEmpty()))`)